pkg/streamingpromql/testdata/ours/functions.test

```diff
@@ -940,3 +940,83 @@ eval range from 0 to 4m step 1m sort_desc(test_metric)
   test_metric{case="histogram with NaN"}  {{count:0 sum:NaN}} {{count:0 sum:NaN}} {{count:0 sum:NaN}} {{count:0 sum:NaN}} {{count:0 sum:NaN}}
   test_metric{case="histogram with +Inf"} {{count:0 sum:Inf}} {{count:0 sum:Inf}} {{count:0 sum:Inf}} {{count:0 sum:Inf}} {{count:0 sum:Inf}}
   test_metric{case="histogram with -Inf"} {{count:0 sum:-Inf}} {{count:0 sum:-Inf}} {{count:0 sum:-Inf}} {{count:0 sum:-Inf}} {{count:0 sum:-Inf}}
+
+clear
+
+# absent() tests
+load 4m
+  series{case="-"} _x11
+  series{case="0a"} _x11
+  series{case="0b"} _x11
+  series{case="0"} 0x11
+  series{case="a"} _ 1 _ 1 _ 1 _ 1 _ 1 _ 
+  series{case="b"} _ _ 2 _ 2 _ 2 _ 2 _ 2 
+  series{case="c"} _ _ _ 3 _ _ _ 3 _ _ _ 
```

Comment:
Could you please add a test for instant and range queries with `absent(series{case=~"(a|b)"})`? This will flush out the changes required in the operator to correctly handle multiple input series.

Reply:
Added in 3765fcf6d007f560ed79ce8c2d19f9d9522e74f7